added textured soda can models

tapomayukh opened the pull request:
new file:   can.bak.urdf
	modified:   can.urdf
	new file:   can_collision.stl
	new file:   can_visual.dae
	new file:   model/__Metal_Aluminum_Anodized_.jpg
	new file:   model/__Metal_Aluminum_Anodized__0.jpg
	new file:   model/coladosentextur.jpg
	new file:   model/coladosentextur_1.jpg

tapomayukh committed:
On branch tapo_demo_urdfs
	new file:   can.bak.urdf
	modified:   can.urdf
	new file:   can_collision.stl
	new file:   can_visual.dae
	new file:   model/__Metal_Aluminum_Anodized_.jpg
	new file:   model/__Metal_Aluminum_Anodized__0.jpg
	new file:   model/coladosentextur.jpg
	new file:   model/coladosentextur_1.jpg

siddhss5 commented:
It’s pretty insane to have a collision model for a cylinder to be that large! It will kill our collision checking speed. Please replace with a primitive cylinder.

siddhss5 commented:
By that I meant a cylinder primitive in DART.

tapomayukh committed:
modified the urdf file to replace the stl
 collision model with a cylinder primitive shape
 Changes to be committed:
	modified:   can.urdf

jslee02 commented on data/objects/can.bak.urdf:
```diff
@@ -0,0 +1,19 @@
+<robot name="can">
```
Do we really need to keep the original can model?

jslee02 commented on data/objects/can.urdf:
```diff
@@ -1,18 +1,15 @@
-      <material name="c">
-        <color rgba="0.65 0.7 0.7 1.0" />
-      </material>
+      <origin xyz="-0.3 0.0 0.0" />
```
I guess this origin is necessary because of the offset of the visual mesh. With this offset, is the center of the visual mesh will be placed at (0, 0, 0) in the local frame?

Also, the origin of collision shape shouldn't be changed because we keep using the primitive cylinder.

jslee02 reviewed (commented):
[`can_collision.stl`](https://github.com/personalrobotics/pr_ordata/blob/223f456c43b0dc5ae5b16af22fdc56c8d5f03079/data/objects/can_collision.stl) seems broken: (1) the top side is out of the main cylinder body (2) there are two weird triangles.

More importantly, I think we don't need a collision mesh file because we keep using a primitive cylinder shape.
